src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -64,6 +64,32 @@ private static VersionInfo GetVersionInfo()
 
         private const string RegistrationName = "CopyColumns";
 
+        public static CopyColumnsTransform Create(IHostEnvironment env, IDataView input, params string[] inputColumns)
+        {
+            var inputOutputColumns = new(string inputColumn, string outputColumn)[inputColumns.Length];
+            for (int i = 0; i < inputColumns.Length; i++)
+            {
+                inputOutputColumns[i].inputColumn = inputOutputColumns[i].outputColumn = inputColumns[i];
+            }
+            return Create(env, input, inputOutputColumns);
```

Comment:
The only practical use of the copy transform is to, effectively, rename columns. Therefore, handling the case where name of the output is identical to that of the input seems to serve no purpose.